airframe-metrics/jvm/src/main/scala/wvlet/airframe/metrics/TimeWindow.scala

```diff
@@ -92,8 +92,9 @@ case class TimeWindow(start: ZonedDateTime, end: ZonedDateTime) {
     }
   }
 
-  def plus(n: Long, unit: ChronoUnit)  = TimeWindow(start.plus(n, unit), end.plus(n, unit))
-  def minus(n: Long, unit: ChronoUnit) = plus(-n, unit)
+  def plus(n: Long, unit: ChronoUnit): TimeWindow  = TimeWindow(start.plus(n, unit), end.plus(n, unit))
+  def minus(n: Long, unit: ChronoUnit): TimeWindow = plus(-n, unit)
+  def between(unit: ChronoUnit): Long              = unit.between(start, end)
```

Comment:
The meaning of `between` is a bit unclear for me. 

How about using this name?
```scala
def howManyOf(unit: ChromeUnits):Long
``` 

or
```scala
def countUnits(unit: ChronoUnit): Long
```

Reply:
```
TimeWindow.howMany(DAYS) 
TimeWindow.howMany(HOURS)
```

Reply:
@xerial Thank your review. I used by ChronoUnit's method name but it's not clear just pointed out and I thought so.

How do you think the name of 'datediff'?  It is usually used by SQL functions method names.

1. `datediff`
2. `howMany`
3. `countUnits `

I like to use 1 or 2.

Reply:
Yeah. I know `ChromeUnit.DAYS.between(s, e)`

If we go with datediff, `def diffIn(unit)` will be better as we can use this for various time units.

I think `howMany(unit)` has the most straightforward meaning.

Reply:
It is also possible to add these functions for convenience:
```
def dateDiff: Long = howMany(DAYS)
def monthDiff: Long = howMany(MONTHS)
...
```

Reply:
@xerial Thanks in your advice. I'll fix and implement the idea in next.